java/src/jmri/DigitalIO.java

```diff
@@ -10,12 +10,12 @@ public interface DigitalIO extends NamedBean {
     /**
      * State value indicating output is on.
      */
-    int ON = 0x02;
+    static final int ON = 0x02;
```

Comment:
What's the benefit of this? I might have cases where I should do it too.

Reply:
The `static` is redundant because data members in a Java interface are static by default.  I think it serves as a nice reminder though.

The final is not the default, so adding that makes this into a true constant that can't be changed inadvertently.

Ideally, some of these small-integer internal constants could be changed into enum values, but that's a task for another day.

Reply:
Thanks.

Maybe a long shot but could the problem with Graal not being able to access Turnout.UNKNOWN but can access Turnout.THROWN be related to these constants being variables and not constants?

Reply:
Good thought!  I tried it and, unfortunately, it didn't fix the problem with inherited constants. Definitely worth the try.